testing/oak_echo_bin/README.md

```diff
@@ -1,4 +1,10 @@
-# Echo Baremetal
+# Oak Echo
 
-Echo baremetal binary that can be run under
+Echo freestanding binary that can be run under
 [crosvm](https://chromium.googlesource.com/chromiumos/platform/crosvm/).
+
+Echo binary receives a message as bytes, and sends back the same bytes, without interpreting them.
```

Comment:
This is true for the raw one, right? This binary is interpreting the bytes via the IDL, which is the difference between them. Or did I misunderstand?

Reply:
Yes, `oak_echo_bin` uses IDL but still receives bytes and sends them back.
Updated the comment.